Sources/Networking/BackendConfiguration.swift

```diff
@@ -15,22 +15,16 @@ import Foundation
 
 class BackendConfiguration {
 
-    let apiKey: String
-    let authHeaders: [String: String]
     let httpClient: HTTPClient
 
     let callbackQueue: DispatchQueue
     let operationQueue: OperationQueue
     let dateProvider: DateProvider
 
-    init(apiKey: String,
-         authHeaders: [String: String],
-         httpClient: HTTPClient,
+    init(httpClient: HTTPClient,
```

Comment:
@NachoSoto I blame you for this commit.... the ripple effect was pretty big, but I think this refactor moves us in the right direction 😄